### PR TITLE
Remove NativeTokens

### LINKED ISSUE #1985: Remove NativeTokens and NativeTokensBuilder?
Doesn't really seem necessary anymore since NT is a Feature. There is not even a count max so it can be replaced by a map when needed.

### PR DESCRIPTION
**Pull request by thibault-martinez**
Fixes https://github.com/iotaledger/iota-sdk/issues/1985

### CODE REVIEW

File: cli/src/wallet_cli/mod.rs [hunk 1]
```diff
@@ -1363,14 +1362,7 @@ async fn print_wallet_address(wallet: &Wallet) -> Result<(), Error> {
 
     log = format!(
         "{log}\nOutputs: {:#?}\nBase coin amount: {}\nNative Tokens: {:?}\nAccounts: {:?}\nFoundries: {:?}\nNFTs: {:?}\nDelegations: {:?}\nAnchors: {:?}\n",
-        output_ids,
```
Comment (DaughterOfMars) [inline cli/src/wallet_cli/mod.rs:1332]: maybe split this up or something ew

Comment (thibault-martinez) [reply cli/src/wallet_cli/mod.rs:1332]: Done

Comment (DaughterOfMars) [reply cli/src/wallet_cli/mod.rs:1332]: Can't we just push the strings rather than the weird formatting thing?

Comment (DaughterOfMars) [reply cli/src/wallet_cli/mod.rs:1332]: Or even just log them one at a time??

**Review by Thoralf-M — DISMISSED** [review]

**Review by Alex6323 — DISMISSED** [review]

**Review by Thoralf-M — DISMISSED** [review]

File: cli/src/wallet_cli/mod.rs (hunk 1 above)
Comment (DaughterOfMars) [reply cli/src/wallet_cli/mod.rs:1332]: pleasssee

**Review by Alex6323 — DISMISSED** [review]

File: cli/src/wallet_cli/mod.rs (hunk 1 above)
Comment (Thoralf-M) [reply cli/src/wallet_cli/mod.rs:1332]: @thibault-martinez want to do this?

**Pull request closed without merging**